trace_decoder/src/hermez_cdk_erigon.rs

```diff
@@ -0,0 +1,244 @@
+//! Frontend for the witness format emitted by [`0xPolygonHermez/cdk-erigon`](https://github.com/0xPolygonHermez/cdk-erigon/)
```

Comment:
I'm not sure we should name the file this. The repo might actually be moved soon so it wouldn't be `hermez...` I think a more generic name is better but that's also a very fussy opinion.

Reply:
I've named the files this way because they reflect vendor-specific support. I'm not fussed by the actual name, but "type1" and "type2" didn't really sit right. Happy to take suggestions

Reply:
`type 1` and `type 2` is the standard nomenclature. It's not just "ah this is v1 so let's call this v2". See [here](https://vitalik.eth.limo/general/2022/08/04/zkevm.html).

Reply:
> I understand, but I think the type2 => hermez and type1 => plonky are polygon-specific aliases.

plonky2 will be both t1 and t2, I mean it already is but t2 is still in a different branch for a little bit longer. Hermez is our existing t2 prover which means we tend to conflate them but strictly speaking it's incorrect to do so. 

> My worry was that calling this module type1 implied that the logic would work for all type1 provers

All of this code will only work with plonky2 not with hermez inc. the t2 stuff.

I vote going type1 and type2.